### PR TITLE
Feature: Interactive List

### PR DESCRIPTION
**Pull request by nikoescobal**
## In this milestone, I did the following:
- Add interactivity for the to-do-list
- Add a new JavaScript file to import as a module
- Add change event listener to checkbox 
- Add store functionality to update the array in local storage, so correct list values are given to the user post-page reload.

### CODE REVIEW

File: src/feature.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
+export const checkIfDone = (done) => !done;
```
Comment (rloterh) [inline src/feature.js:2]: Your module file is **incomplete**.  It should contain the following:

- [ ] Methods related to the status updates (completed: true / false).

File: src/feature.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (rloterh) [inline src/feature.js:1]: - [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

File: dist/index.html [hunk 3]
```diff
@@ -13,12 +13,12 @@
 <body>
```
Comment (rloterh) [inline dist/index.html:13]: - [ ] OPTIONAL
Your application is not able to run using the command `npm start`. Kindly add the necessary _scripts options_ to correct this issue.

Error generated on the command `npm start`:
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Niko, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/feature.js (hunk 2 above)
Comment (nikoescobal) [reply src/feature.js:1]: > * [ ]  You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Hi. Thanks for the review. I would get just one specific eslint error pertaining to the import syntax, which is why I disabled it. I think that's reasonable.

File: dist/index.html (hunk 3 above)
Comment (nikoescobal) [reply dist/index.html:13]: The reason your script isn't working is that I don't have npm start in my package.json. You should have checked so you would know to run `npm run dev` or `npm run build.`

File: src/feature.js (hunk 1 above)
Comment (nikoescobal) [reply src/feature.js:2]: The status update is explicitly reflected by how a task item can be marked as done or not done, as seen in how the strikethrough being is applied. The status of isDone literally functions the same way as isCompleted, meaning it updates the status just fine.

**Review by carlos-ssh — APPROVED** [review]
Hi @nikoescobal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Temmarie** [comment]
Hi @nikoescobal 👋🏾 

_This is an additional review as a result of the second opinion request._

Seeing as the issue has been resolved, I'll go ahead and close the request. Have a lovely day!
Happy coding and keep rocking! 🚀

**Pull request merged**